app/schema.py

```diff
@@ -12,13 +12,38 @@ class Meta:
         model = User
 
 
+class FriendRequestType(DjangoObjectType):
+    class Meta:
+        model = FriendRequest
+
+
+class FriendType(DjangoObjectType):
+    class Meta:
+        model = Friend
+
+
 class Query(ObjectType):
     user = graphene.Field(UserType, contact=graphene.String())
+    friend_requests_sent = graphene.List(FriendRequestType)
+    friend_requests_received = graphene.List(FriendRequestType)
+    friends = graphene.List(FriendType)
 
     @login_required
     def resolve_user(self, info, **kwargs):
         return info.context.user
 
+    @login_required
+    def resolve_friend_requests_sent(parent, info, **kwargs):
```

Comment:
Change parent to self. By PEP standards, first parameter of a method should be named self